tests/driver_lis2dh12/main.c

```diff
@@ -48,6 +73,23 @@ int main(void)
         return 1;
     }
 
+    /* enable interrupt Pins */
+    char* pin = "PA12";
+    if(gpio_init_int(GPIO_PIN(PA,12),GPIO_IN, GPIO_RISING,int_cb,pin) == -1)
```

Comment:
Compiling this code makes the assumption that the lis2dh12 interrupts are wired to the GPIO_PIN(PA, 12) and GPIO_PIN(PA, 13).

Reply:
I added a TODO firstly. Can be changed later...